src/routes/curator-actions/batch/everything/+page.svelte

```diff
@@ -101,180 +105,336 @@
-				Currently, the child bounty’s index is estimated by incrementing the highest available on
-				the blockchain. To create multiple bounties in one batch transaction, Bounty Manager
-				increments this index by 1 for each additional bounty. <br /> Please note: if multiple child
-				bounties are assigned the same index, or if another bounty creates a child between this transaction’s
-				creation and confirmation, this transaction will fail.
-			</p>
-
-			<label class="block w-full sm:max-w-40">
-				<span class="text-xs block">Starting Child Bounty Index</span>
-				<input
-					type="number"
-					min={nextAvailableChildBountyId}
-					bind:value={childBountyId}
-					class="border border-black rounded-[3px] bg-white pl-2 pt-1 h-10 w-full"
-					required
-				/>
-			</label>
-
-			<div class="grid cardsGrid gap-6">
-				{#each childBounties as child, index (child)}
-					<fieldset class="bg-white -mt-3 p-5 lg:w-full rounded-md shadow-lg">
-						<legend class="relative top-7">Child bounty #{index + 1}</legend>
-
-						<div class="flex flex-col gap-6 mt-6">
-							<label>
-								<span class="text-xs block">Value</span>
-								<input
-									bind:value={child.value}
-									class={Input.polkadot}
-									placeholder="00.00"
-									required
-									oninput={validateBountyValue}
-									inputmode="decimal"
-								/>
-							</label>
-
-							<label>
-								<span class="text-xs block">Title</span>
-								<input
-									bind:value={child.title}
-									class={Input.input}
-									placeholder="Child bounty name"
-									required
-								/>
-							</label>
-
-							<label>
-								<span class="text-xs block">Sub-curator fee</span>
-								<input
-									bind:value={child.fee}
-									class={Input.polkadot}
-									placeholder="00.00"
-									required
-									oninput={validateFee}
-									inputmode="decimal"
-								/>
-							</label>
-
-							<label>
-								<span class="text-xs block">Beneficiary account address</span>
+			<form onsubmit={submit} class="form">
+				<section>
+					<h2 class="headerForm">Complete Payout</h2>
+				</section>
+				<div class="contentForm">
+					<section class="stepsList">
+						<h3 class="textContent">Batch all child bounty operations in one transaction:</h3>
+
+						<ol class="textContent">
+							<li>Create a new child bounty</li>
+							<li>Assign the curator proxy as sub-curator</li>
+							<li>Accept the sub-curator role</li>
+							<li>Award the child bounty to the provided beneficiary</li>
+							<li>Claim the child bounty (optional)</li>
+						</ol>
+					</section>
+
+					<div class="cardsGrid">
+						{#each childBounties.concat( { value: '', title: '', fee: '', beneficiary: '', claim: false, template: true } ) as child, index (child)}
+							<fieldset class="card">
+								<div class={child.template ? 'blurred' : ''}>
+									<legend class="cardIndex"
+										>Child Bounty <span class="index">{index + 1}</span></legend
+									>
```

Comment:
```suggestion
									<legend class="cardIndex">
										Child Bounty <span class="index">{index + 1}</span>
									</legend>
```
Let’s fix the formatting